docs/static/releasenotes.asciidoc

```diff
@@ -55,6 +56,43 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-13-0]]
+=== Logstash 8.13.0 Release Notes
+
+* Package `elastic_integration` plugin. https://github.com/elastic/logstash/pull/15769[#15769]
+* Flow metrics added `worker_utilization` to pipeline-level https://github.com/elastic/logstash/pull/15912[#15912]
+
+==== Plugins
+
+*Elastic_integration Filter - 0.1.6*
```

Comment:
```suggestion
*Elastic Integration Filter - 0.1.6*
```

OR
```suggestion
*`elastic_integration` Filter - 0.1.6*
```

Reply:
I think the name has to follow the convention with underscore like [Elastic_serverless_forwarder](https://github.com/elastic/logstash/pull/15265#discussion_r1312424994)